P5/Source/Guidelines/en/CO-CoreElements.xml

```diff
@@ -1935,6 +1935,12 @@ avoid the need to repeat all the normalizing attributes:
     <measure type="list"><num>1</num>, <num>2</num>, <num>5</num>, <num>7</num>
       <unit type="length" norm="mm">millimètres</unit></measure>
```

Comment:
the word "millimètres" is still in French (and it says @xml:lang="en" in `egXML`)